CONTRIBUTING.md

```diff
@@ -44,7 +44,7 @@ Building JupyterLab from its GitHub source code requires Node.js version
 If you use `conda`, you can get it with:
 
 ```bash
-conda install -c conda-forge nodejs
+conda install -c conda-forge nodejs=9.11.1
```

Comment:
Thanks. Can we make this just `nodejs<10`? I think that conveys the idea and I think it is still correct.